recipes/napari-locan/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "napari-locan" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-locan-{{ version }}.tar.gz
+  sha256: 476619faccd96d847a17ddb82634b908f754bcd4d0c01af3b379f2b9d902551f
+
+build:
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```

Comment:
Could we sort all the requirements alphabetically. It helps with maintenance and readability :)